### PR TITLE
Minor: Expose new fields in /contents API. RD-19314

### PR DESCRIPTION
**Pull request by alexeykondratenko**

### CODE REVIEW

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -6746,6 +6753,9 @@ components:
           type: boolean
         published:
           type: boolean
+        rating:
+          description: is present only if the content supports rating and raiting is filled.
```
Comment (ArtyomIvanov-dev) [inline specs/engage-digital_openapi3.yaml:6757]: What 'raiting' stands for? What does it mean?

Comment (alexeykondratenko) [reply specs/engage-digital_openapi3.yaml:6757]: Not sure that I understand your question. It's filed in ED on some sources, I suppose you should check internal MR for more details.

Comment (alexeykondratenko) [reply specs/engage-digital_openapi3.yaml]: I fixed typo.

**Comment by ArtyomIvanov-dev** [comment]
LGTM

**Pull request merged**